segmentation/include/pcl/segmentation/ground_plane_comparator.h

```diff
@@ -213,14 +213,15 @@ namespace pcl
       {
         // Normal must be similar to neighbor
         // Normal must be similar to expected normal
-        float threshold = distance_threshold_;
-        if (depth_dependent_)
-        {
-          Eigen::Vector3f vec = (*input_)[idx1].getVector3fMap ();
+        // TODO check logic in this class: which member variables are needed?
```

Comment:
Open an issue rather than a note that will be forgotten 😀 ?

Reply:
Fair point, there is already issue https://github.com/PointCloudLibrary/pcl/issues/5099 to remind us to check the logic in the class

Reply:
Yeah, thats fine 😀